runtime/doc/lua.txt

```diff
@@ -968,16 +968,35 @@ paste({lines}, {phase})                                          *vim.paste()*
                     |paste|
 
 schedule_wrap({cb})                                      *vim.schedule_wrap()*
-                Defers callback `cb` until the Nvim API is safe to call.
+                Wraps callback `cb` in a function which will defer the execution
+                of `cb` until the Nvim API is safe to call.
+
+                The user must call the returned function themselves if it is not
+                being used as an argument to a function that will call it for them,
+                such as |nvim_buf_attach()|.
```

Comment:
I think you could say something more along the lines of: "{cb} is not called after being wrapped. To automatically call, use |vim.schedule()|"

Or something like that?

I think this is too wordy.

Reply:
I guess I'm trying to say, if you're doing a one-off wrap & call, you can just use `vim.schedule`. If you want to keep the function always safe to call, then you should use `vim.schedule_wrap`. 

I'm too tired to think of a good way to say it thought.